rules/default.js

```diff
@@ -13,9 +16,13 @@ module.exports = {
                 content : code
             };
 
+            if (tag) {
+                result.tag = 'code';
+            }
+
             if (lang) {
-                result.mods  = {
-                    lang : 'lang'
+                result.mods  = {  // BUG: MUST BE elemMods
```

Comment:
I fixed this bugs already in this commit:

https://github.com/vithar/markdown-bemjson/commit/7673eabd7fb7c78b7cbb5b90ae4a5980c7d9ddfd

I'll send another PR after this will be merged.